code-compass.el

```diff
@@ -890,11 +890,22 @@ code can infer it automatically."
   "Show pie chart of CSV FILE."
   (format c/pie-or-bar-chart-command file))
 
-(defun c/show-fragmentation-sync (file &optional date)
-  "Show knowledge fragmentation for FILE."
+(defun c/sum-by-first-column (rows)
```

Comment:
Very cool and efficient! (I think this function is so useful in general that there should be a built-in!)